src/js/components/Tip/__tests__/__snapshots__/Tip-test.js.snap

```diff
@@ -204,30 +204,6 @@ exports[`Tip focus and blur events on the Tip's child 1`] = `
   -webkit-font-smoothing: antialiased;
 }
 
-@media only screen and (max-width:768px) {
```

Comment:
Any thoughts on why these lines were removed?

Reply:
I'm not sure why they were removed in this PR, but seems like they shouldn't have been there in the first place. Related issue in styled components where empty media queries are added to snapshots: https://github.com/styled-components/jest-styled-components/issues/276